drivers/platform/maxim/max32670/maxim_spi.c

```diff
@@ -57,6 +57,38 @@
 /************************ Functions Definitions *******************************/
 /******************************************************************************/
 
+/**
+ * @brief Configure the VDDIO level for a SPI interface
+ * @param desc - the SPI descriptor
+ * @return 0 in case of success, -EINVAL otherwise
+ */
+static int32_t _max_spi_config_pins(struct no_os_spi_desc *desc)
+{
+	struct max_spi_init_param *eparam;
+	mxc_gpio_cfg_t spi_pins;
+
+	eparam = desc->extra;
+
+	switch(desc->device_id) {
+	case 0:
+		spi_pins = gpio_cfg_spi0;
+		break;
+	case 1:
+		spi_pins = gpio_cfg_spi1;
+		break;
+	case 2:
+		spi_pins = gpio_cfg_spi2;
+		break;
+	default:
+		return -EINVAL;
+	}
+
+	spi_pins.vssel = eparam->vssel;
+	MXC_GPIO_Config(&spi_pins);
+
+	return 0;
+}
```

Comment:
May be worth mentioning in the commit description that MAX32670 only has 1 CS per SPI instance (which is already included in the `gpio_cfg_spi` pin config structs), so we don't need to configure it independently, as for the other Maxim drivers.